[2021-12-22 21:19:19,110] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.super_resolution scheduled__2021-12-20T00:58:00+00:00 [queued]>
[2021-12-22 21:19:19,113] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.super_resolution scheduled__2021-12-20T00:58:00+00:00 [queued]>
[2021-12-22 21:19:19,113] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 21:19:19,113] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-22 21:19:19,114] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 21:19:19,122] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): super_resolution> on 2021-12-20 00:58:00+00:00
[2021-12-22 21:19:19,124] {standard_task_runner.py:52} INFO - Started process 48629 to run task
[2021-12-22 21:19:19,125] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline1', 'super_resolution', 'scheduled__2021-12-20T00:58:00+00:00', '--job-id', '347', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmpqhk2cntu', '--error-file', '/tmp/tmp6_jvm5zs']
[2021-12-22 21:19:19,126] {standard_task_runner.py:77} INFO - Job 347: Subtask super_resolution
[2021-12-22 21:19:19,144] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline1.super_resolution scheduled__2021-12-20T00:58:00+00:00 [running]> on host myamya
[2021-12-22 21:19:19,178] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline1
AIRFLOW_CTX_TASK_ID=super_resolution
AIRFLOW_CTX_EXECUTION_DATE=2021-12-20T00:58:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-20T00:58:00+00:00
[2021-12-22 21:19:19,179] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-22 21:19:19,180] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ sure:0.3 /bin/bash /home/myamya/project/init.sh ']
[2021-12-22 21:19:19,194] {subprocess.py:85} INFO - Output:
[2021-12-22 21:19:20,867] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /opt/conda/conda-bld/pytorch_1603729096996/work/c10/cuda/CUDAFunctions.cpp:100.)
[2021-12-22 21:19:20,867] {subprocess.py:89} INFO -   return torch._C._cuda_getDeviceCount() > 0
[2021-12-22 21:19:21,865] {subprocess.py:89} INFO - Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.1.0/detection_Resnet50_Final.pth" to /opt/conda/lib/python3.8/site-packages/facexlib/weights/detection_Resnet50_Final.pth
[2021-12-22 21:19:21,865] {subprocess.py:89} INFO - 
[2021-12-22 21:19:25,142] {subprocess.py:89} INFO -   0% 0.00/104M [00:00<?, ?B/s]  0% 104k/104M [00:00<02:00, 909kB/s]  0% 520k/104M [00:00<01:32, 1.17MB/s]  2% 2.14M/104M [00:00<01:05, 1.63MB/s]  5% 5.13M/104M [00:00<00:46, 2.25MB/s]  9% 9.02M/104M [00:00<00:32, 3.10MB/s] 12% 12.1M/104M [00:00<00:22, 4.25MB/s] 16% 16.6M/104M [00:00<00:15, 5.84MB/s] 20% 20.6M/104M [00:00<00:11, 7.88MB/s] 24% 24.8M/104M [00:01<00:08, 10.4MB/s] 28% 29.3M/104M [00:01<00:05, 13.6MB/s] 32% 33.4M/104M [00:01<00:04, 17.1MB/s] 36% 37.8M/104M [00:01<00:03, 21.1MB/s] 40% 42.2M/104M [00:01<00:02, 25.2MB/s] 44% 46.4M/104M [00:01<00:02, 28.9MB/s] 49% 50.9M/104M [00:01<00:01, 32.7MB/s] 53% 55.3M/104M [00:01<00:01, 35.8MB/s] 57% 59.6M/104M [00:01<00:01, 38.0MB/s] 62% 64.4M/104M [00:01<00:01, 40.8MB/s] 66% 68.8M/104M [00:02<00:00, 42.2MB/s] 70% 73.2M/104M [00:02<00:00, 43.1MB/s] 75% 78.0M/104M [00:02<00:00, 45.0MB/s] 79% 82.5M/104M [00:02<00:00, 45.3MB/s] 83% 86.9M/104M [00:02<00:00, 45.5MB/s] 88% 91.8M/104M [00:02<00:00, 46.9MB/s] 92% 96.3M/104M [00:02<00:00, 46.9MB/s] 97% 101M/104M [00:02<00:00, 46.3MB/s] 100% 104M/104M [00:02<00:00, 38.5MB/s]
[2021-12-22 21:19:25,276] {subprocess.py:89} INFO - Downloading: "https://github.com/TencentARC/GFPGAN/releases/download/v0.2.0/GFPGANCleanv1-NoCE-C2.pth" to /opt/conda/lib/python3.8/site-packages/gfpgan/weights/GFPGANCleanv1-NoCE-C2.pth
[2021-12-22 21:19:25,277] {subprocess.py:89} INFO - 
[2021-12-22 21:19:33,036] {subprocess.py:89} INFO -   0% 0.00/332M [00:00<?, ?B/s]  0% 200k/332M [00:00<02:50, 2.04MB/s]  0% 1.05M/332M [00:00<02:11, 2.65MB/s]  1% 3.07M/332M [00:00<01:36, 3.59MB/s]  2% 8.28M/332M [00:00<01:08, 4.99MB/s]  4% 13.5M/332M [00:00<00:48, 6.86MB/s]  6% 18.8M/332M [00:00<00:35, 9.31MB/s]  7% 24.3M/332M [00:00<00:26, 12.4MB/s]  9% 29.7M/332M [00:00<00:19, 16.2MB/s] 11% 35.2M/332M [00:00<00:15, 20.7MB/s] 12% 40.7M/332M [00:01<00:11, 25.6MB/s] 14% 46.3M/332M [00:01<00:09, 30.8MB/s] 16% 51.9M/332M [00:01<00:08, 35.9MB/s] 17% 57.2M/332M [00:01<00:07, 38.5MB/s] 19% 62.3M/332M [00:01<00:07, 40.2MB/s] 20% 67.1M/332M [00:01<00:06, 40.8MB/s] 22% 71.7M/332M [00:01<00:06, 42.0MB/s] 23% 76.2M/332M [00:01<00:06, 42.0MB/s] 24% 80.5M/332M [00:01<00:06, 42.8MB/s] 26% 84.9M/332M [00:01<00:06, 42.8MB/s] 27% 89.2M/332M [00:02<00:05, 43.4MB/s] 28% 93.4M/332M [00:02<00:05, 43.3MB/s] 29% 97.9M/332M [00:02<00:05, 44.1MB/s] 31% 102M/332M [00:02<00:05, 43.8MB/s]  32% 107M/332M [00:02<00:05, 44.6MB/s] 33% 111M/332M [00:02<00:05, 44.5MB/s] 35% 115M/332M [00:02<00:05, 44.9MB/s] 36% 120M/332M [00:02<00:04, 45.2MB/s] 37% 124M/332M [00:02<00:04, 45.3MB/s] 39% 129M/332M [00:03<00:04, 45.9MB/s] 40% 133M/332M [00:03<00:04, 45.6MB/s] 41% 137M/332M [00:03<00:04, 46.3MB/s] 43% 142M/332M [00:03<00:04, 46.1MB/s] 44% 146M/332M [00:03<00:04, 46.6MB/s] 45% 151M/332M [00:03<00:04, 46.9MB/s] 47% 155M/332M [00:03<00:03, 46.8MB/s] 48% 160M/332M [00:03<00:03, 47.4MB/s] 50% 165M/332M [00:03<00:03, 47.0MB/s] 51% 169M/332M [00:03<00:03, 47.5MB/s] 52% 174M/332M [00:04<00:03, 47.4MB/s] 54% 179M/332M [00:04<00:03, 47.8MB/s] 55% 183M/332M [00:04<00:03, 47.9MB/s] 56% 188M/332M [00:04<00:03, 48.0MB/s] 58% 192M/332M [00:04<00:03, 48.3MB/s] 59% 197M/332M [00:04<00:02, 48.2MB/s] 61% 202M/332M [00:04<00:02, 48.6MB/s] 62% 206M/332M [00:04<00:02, 48.8MB/s] 64% 211M/332M [00:04<00:02, 48.7MB/s] 65% 216M/332M [00:04<00:02, 49.1MB/s] 66% 221M/332M [00:05<00:02, 48.8MB/s] 68% 225M/332M [00:05<00:02, 49.3MB/s] 69% 230M/332M [00:05<00:02, 49.1MB/s] 71% 235M/332M [00:05<00:02, 49.4MB/s] 72% 240M/332M [00:05<00:01, 49.4MB/s] 74% 244M/332M [00:05<00:01, 49.6MB/s] 75% 249M/332M [00:05<00:01, 50.0MB/s] 76% 254M/332M [00:05<00:01, 49.7MB/s] 78% 259M/332M [00:05<00:01, 50.0MB/s] 79% 264M/332M [00:05<00:01, 49.9MB/s] 81% 268M/332M [00:06<00:01, 50.0MB/s] 82% 273M/332M [00:06<00:01, 50.3MB/s] 84% 278M/332M [00:06<00:01, 50.2MB/s] 85% 283M/332M [00:06<00:01, 49.2MB/s] 87% 288M/332M [00:06<00:00, 49.6MB/s] 88% 293M/332M [00:06<00:00, 49.7MB/s] 89% 297M/332M [00:06<00:00, 50.2MB/s] 91% 302M/332M [00:06<00:00, 49.9MB/s] 92% 307M/332M [00:06<00:00, 50.4MB/s] 94% 312M/332M [00:06<00:00, 50.4MB/s] 95% 317M/332M [00:07<00:00, 50.4MB/s] 97% 322M/332M [00:07<00:00, 50.7MB/s] 98% 327M/332M [00:07<00:00, 50.5MB/s]100% 332M/332M [00:07<00:00, 50.8MB/s]100% 332M/332M [00:07<00:00, 47.4MB/s]
[2021-12-22 21:19:33,197] {subprocess.py:89} INFO - Testing 0 2271
[2021-12-22 21:19:33,301] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/nn/functional.py:3103: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details.
[2021-12-22 21:19:33,302] {subprocess.py:89} INFO -   warnings.warn("The default behavior for interpolate/upsample with float scale_factor changed "
[2021-12-22 21:19:37,271] {subprocess.py:89} INFO - Testing 1 2272
[2021-12-22 21:19:41,927] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-22 21:19:41,966] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline1, task_id=super_resolution, execution_date=20211220T005800, start_date=20211222T121919, end_date=20211222T121941
[2021-12-22 21:19:42,008] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-22 21:19:42,043] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
